hicommonwealth/commonwealth
# Prevent filter row redraw on thread filter change

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/4156

## Description of Changes
- Filter row wont re-render when thread filter changes 
- The cow bobber now shows below the thread filter row when the threads are loading

## Test Plan
Change different filters on the discussion page 
- Verify the filter row does not re-render
- Verify the cow bobber shows below filter row

## Deployment Plan
N/A

## Other Considerations
N/A
